lib/galaxy/schema/cloud.py

```diff
@@ -0,0 +1,160 @@
+from typing import (
+    Any,
+    List,
+    Optional,
+    Union,
+)
+
+from pydantic import (
+    Field,
+    Required,
+)
+from typing_extensions import Literal
+
+# from galaxy.model import Dataset
+from galaxy.schema.fields import DecodedDatabaseIdField
+from galaxy.schema.schema import Model
+
+
+class InputArguments(Model):
+    dbkey: Optional[str] = Field(
+        default="?",
+        title="Genome",
+        description="Sets the genome (e.g., `hg19`) of the objects being fetched to Galaxy.",
+    )
```

Comment:
```suggestion
    dbkey: Optional[str] = Field(
        default="?",
        title="Database Key",
        description="Sets the database key of the objects being fetched to Galaxy.",
    )
```
If I remember correctly, we are trying to move away from the genome concept associated with the `dbkey` as it could be something more generic.